clients/js/near.ts

```diff
@@ -1,10 +1,112 @@
 import { impossible, Payload } from "./vaa";
 import { NETWORKS } from "./networks";
 import { CONTRACTS } from "@certusone/wormhole-sdk";
+const { parseSeedPhrase, generateSeedPhrase } = require("near-seed-phrase");
+const fs = require("fs");
 
 const BN = require("bn.js");
 const nearAPI = require("near-api-js");
 
+function default_near_args(argv) {
+  let network = argv["n"].toUpperCase();
+  let contracts = CONTRACTS[network]["near"];
+  let n = NETWORKS[network]["near"];
+
+  if (!("rpc" in argv)) {
+    argv["rpc"] = n["rpc"];
```

Comment:
Since rpc is commented out in main.ts, isn't this always the case?

Reply:
the rpc can come from a higher level argument setting..